lib/influxdb2/client/client.rb

```diff
@@ -57,8 +58,8 @@ def initialize(url, token, options = nil)
     # Write time series data into InfluxDB thought WriteApi.
     #
     # @return [WriteApi] New instance of WriteApi.
-    def create_write_api(write_options: InfluxDB2::SYNCHRONOUS)
-      write_api = WriteApi.new(options: @options, write_options: write_options)
+    def create_write_api(write_options: InfluxDB2::SYNCHRONOUS, point_settings: InfluxDB2::PointSettings.new)
```

Comment:
Please add a static default value for `point_settings` to avoid creating new instance every time.

```suggestion
    def create_write_api(write_options: InfluxDB2::SYNCHRONOUS, point_settings: InfluxDB2::DEFAULT_POINT_SETTINGS)
```